notifications.yaml

```diff
@@ -0,0 +1,220 @@
+        type: "string"
+        minLength: 1
+        maxLength: 255
+        description: "The title of the message. At most 255 characters."
+      body_text: 
+        type: "string"
+        x-faker: "lorem.paragraph"
+      body_html: 
+        type: "string"
+        x-faker: "lorem.paragraph"
+    required: 
+      - "title"
+  MessageDefinition: 
+    title: "MessageDefinition"
+    type: "object"
+    properties: 
+      to: 
+        description: "One fiscal code (CF).\n"
+        type: "string"
+      dry_run: 
+        type: "boolean"
+        description: "This parameter, when set to true, allows developers to test a request without actually sending a message.\n\nThe default value is false."
+      time_to_live: 
+        type: "integer"
+        description: "This parameter specifies how long (in seconds) the message should be kept in storage if the device is offline. The maximum time to live supported is 1 year, and the default value is 1 hour."
+        minimum: 3600
+        maximum: 31536000
+      tags: 
+        type: "array"
+        default: []
+        items: 
+          type: "string"
+      content: 
+        $ref: "#/definitions/MessageContent"
+    required: 
+      - "to"
+      - "content"
+  MessageStatus: 
+    title: "MessageStatus"
+    type: "object"
+    properties: 
+      status: 
+        type: "string"
+        description: "ACCEPTED: the message has been accepted for delivery\n\nDELIVERED: the message was delivered to the channel\n\nFAILED: something wong happened, check the error code and message\n\nTIMED_OUT:  in case a TTL was set\n"
+        enum: 
+          - "ACCEPTED"
+          - "DELIVERED"
+          - "TIMED_OUT"
+          - "FAILED"
+      failed_at: 
+        type: "string"
+        format: "date-time"
+      delivered_at: 
+        type: "string"
+        format: "date-time"
+      accepted_at: 
+        type: "string"
+        format: "date-time"
+      timed_out_at: 
+        type: "string"
+        format: "date-time"
+      problem: 
+        $ref: "#/definitions/ProblemJson"
+  Message: 
+    title: "Message"
+    allOf: 
+      - 
+        type: "object"
+        properties: 
+          id: 
+            type: "string"
+            format: "uuid"
+          created_at: 
+            type: "string"
+            format: "date-time"
+          status: 
+            type: "object"
+            default: {}
+            properties: 
+              mail: 
+                $ref: "#/definitions/MessageStatus"
+              mobile: 
+                $ref: "#/definitions/MessageStatus"
+              web: 
+                $ref: "#/definitions/MessageStatus"
+        required: 
+          - "id"
+      - 
+        $ref: "#/definitions/MessageDefinition"
+  MessageCollection: 
+    title: "MessageCollection"
+    type: "object"
+    properties: 
+      resources: 
+        type: "array"
+        default: []
+        items: 
+          $ref: "#/definitions/Message"
+      totalResults: 
```

Comment:
we can use HATEOAS semantic like described here: https://zalando.github.io/restful-api-guidelines/pagination/Pagination.html (so we can use 'items'). 

totalResults field was included in coherence with the SCIM API that returns list of resources, but we can switch to HATEOAS here.